### ISSUE TITLE
GE stops responding when clicking the "Commit" button

### ISSUE THREAD

**Issue by iGrimpe** [gitextensions/gitextensions#8233]
## Current behaviour
After opening GE, clicking the "Commit" button to review changes, causes GE to stop responding. the same problem as described here [https://github.com/gitextensions/gitextensions/issues/6427](https://github.com/gitextensions/gitextensions/issues/6427)
after that is FE not responsive, you can click whereever you want and nothing happens

## Expected behaviour
Commit window opened (Should open the "View Changes" dialog)

## Steps to reproduce

Start GE, select repository, click on commit button iin the "Toolbar" on the top of window and... GE doesn't respond anymore

## Did this work in previous version of GitExtensions
Yes/No

It was working befor in version 3.0.2 till some date(X), after that it was not working anymore. I have upgraded the installed version to 3.3.1 but the problem persists.
(X) on that date our partner, where GIT repositorium is hosted, has switched all their applications, available through Internet, to 2-Factor-Authentication. BUT their GIT Server was already having 2-Factor-Authentication befor this date (!!!!!!)

## Environment

- GitExtensions version: 3.3.1
- GIT version: 
- OS version: 
- .NET version:

**Comment by RussKie** [comment]
The commit forms gets opened but outside the visible area, check https://github.com/gitextensions/gitextensions/issues/6427 for workarounds.

**Comment by iGrimpe** [comment]
Hello Igor,
thank you very much fpr your response. The problem was, that commit window was hidden and could not be found (see screenshot).
After getting your response I tried to move unvisible window with keyboard and only this way I was able to find it.
Thank you very much, Irina
![P47043 - Desktop Viewer 17 06 2020 13_51_40](https://user-images.githubusercontent.com/66993946/84895889-5165b800-b0a3-11ea-9daf-3959120f2f95.png)
